core/storage/user/gae_models.py

```diff
@@ -1166,11 +1166,16 @@ def export_data(user_id: str) -> Dict[str, Union[List[str], float, None]]:
 
         # Ruling out the possibility of None for mypy type checking.
         assert user_model is not None
-        creator_user_models = cast(
-            List[UserSettingsModel],
-            UserSettingsModel.get_multi(user_model.creator_ids))
+        creator_user_models = (
+            UserSettingsModel.get_multi(user_model.creator_ids)
+        )
+        filtered_creator_user_models = []
+        for creator_user_model in creator_user_models:
+            # Ruling out the possibility of None for mypy type checking.
```

Comment:
Same thing, needs more justification.

Reply:
assert is removed and exception is added